tests/functional/conf.py

```diff
@@ -45,6 +45,7 @@
 REGISTRY = f"{os.environ.get('IP') or 'localhost'}:5044"
 REGISTRY_TLS = f"{os.environ.get('IP') or 'localhost'}:5045"
 REGISTRY_TLS_INSECURE = f"{os.environ.get('IP') or 'localhost'}:5046"
+REGISTRY_PROXY = f"{os.environ.get('IP') or 'localhost'}:5047"
```

Comment:
is there a reason not to pass the default value to the `.get(key, default_val)` method?

REGISTRY_PROXY = f"{os.environ.get('IP', 'localhost'}:5047"

Reply:
that env var is defaulted in compose file to empty string

https://github.com/crashappsec/chalk/blob/a988c7b4a85608fe5a000cce999827dba0cd4d8e/docker-compose.yml#L159-L159

when not using `make` to invoke `docker compose` which sets `IP`:

https://github.com/crashappsec/chalk/blob/a988c7b4a85608fe5a000cce999827dba0cd4d8e/Makefile#L103-L103

its value is going to be empty string hence `.get()` will return empty string which wont be correct

with `or` it gets correctly defaulted:

```
➜ docker compose run --rm --no-deps --entrypoint=python tests -c 'import conf; print(conf.REGISTRY_PROXY)'
localhost:5047
```